toucan_connectors/awsathena/awsathena_connector.py

```diff
@@ -55,7 +55,9 @@ def __init__(self, **data):
         super().__init__(**data)
         # Named parameters need to be passed as `:name`
         # (see https://aws-data-wrangler.readthedocs.io/en/stable/stubs/awswrangler.athena.read_sql_query.html)
-        self.query, self.parameters = sanitize_query(self.query, self.parameters, lambda x: f':{x}')
+        self.query, self.parameters = sanitize_query(
+            self.query, self.parameters, lambda x: f':{x};'
```

Comment:
Maybe we should convert this lambda into a function with proper docstring.
Tomorrow it will be in weaverbird in a proper API